files/index.html

```diff
@@ -25,13 +25,26 @@ <h3>step 2</h3>
       </div>
 
       <div class="manual-content">
-            <h3>step 3</h3>
+        <img src="https://drive.google.com/uc?export=download&id=1WAe4EvOFEnboWQAvhcu_Mzm1zlDl-KNR" alt="">
```

Comment:
this link doesn't seem to work

Reply:
The privacy setting of the image is updated and should have fixed the problem